colin-api/requirements.txt

```diff
@@ -6,6 +6,7 @@ Flask-SQLAlchemy
 Flask-Marshmallow
 Flask-OIDC
 flask-jwt-oidc
+Werkzeug==0.16.0
```

Comment:
can remove this, should be the newer one now

Reply:
we'll open a techdeb tix for these

Reply:
I tried other version of Wekzeug but all get error. And also tried upgrade the flask-jwt-oidc to newest version. Same, not sure which library had issue.